packages/service-integration/src/service_integration/commands/run_creator.py

```diff
@@ -54,9 +56,7 @@ def main(
             filename = input_key
             if "fileToKeyMap" in input_value and len(input_value["fileToKeyMap"]) > 0:
                 filename, _ = next(iter(input_value["fileToKeyMap"].items()))
-            input_script.append(
-                f"{str(input_key).upper()}=$INPUT_FOLDER/{str(filename)}"
-            )
+            input_script.append(f"{str(input_key).upper()}=$INPUT_FOLDER/{filename!s}")
```

Comment:
isn't this overkill? filename should already be a string

Reply:
```
>>> filename=Path().resolve()
>>> print(f"{filename!s}")
/home/crespo/repos/osparc-simcore/packages/models-library
>>> print(f"{filename}")
/home/crespo/repos/osparc-simcore/packages/models-library
```

It does nto make a big difference

Reply:
I found the change strange, was wondering why, now I understand what triggered it